_sass/color_schemes/light.scss

```diff
@@ -0,0 +1,208 @@
+// Moved from _sass/code.scss
+
+.highlight .c {
+  color: #586e75;
+} // comment //
```

Comment:
Why do we need to abbreviate here? 
There is probably nothing to be gained by shortering "comment" to "c", "name" to "n", and so on. It just makes the CSS harder to read, forcing us to have comments to excuse our naming :)

Reply:
These are a part of a syntax highlighter; we don't define the class names or write this code by hand, we're overriding classes that are inserted by [rouge](https://github.com/rouge-ruby/rouge).